HostsFileUpdaterScript

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Hosts file updater
+# You will probably need root or su rights to access /etc/hosts
+# modified by Valkiry to be used withb Steven Black's Unified Hosts files.
+#
+# Debian users will need the package "sysutils" as this script uses dos2unix
+# $ apt-get install sysutils
+#
+HFSERVER="https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts"
+HFILE="hosts.txt"
+ORIGFILE="/etc/hosts.original"
+
+clear
+echo "-------------------------------------------------------------"
+echo "This script will update your Hosts file to the latest version of StevenBlack's Unified hosts = (adware + malware)"
+echo "Your original Hosts file will be renamed to $ORIGFILE"
+echo "-------------------------------------------------------------"
+echo ""
+
+if [ ! -f "$ORIGFILE" ] ; then
```

Comment:
Understood so for now as it is, I don't think that it's gonna be merged. 
Can't wait to see your update :+1: